assigner.py

```diff
@@ -387,9 +403,9 @@ def main():
     logger.debug("Selected modules to run: {}".format(modules))
 
     if "rra" in modules:
-        autoassign(bapi, capi, config["bugzilla"]["rra"], args.dry_run)
+        autoassign(bapi, capi, config["bugzilla"]["rra"], config["casa"], config["foxsec"], args.dry_run)
     if "va" in modules:
-        autoassign(bapi, capi, config["bugzilla"]["va"], args.dry_run)
+        autoassign(bapi, capi, config["bugzilla"]["va"], config["casa"], config["foxsec"], args.dry_run)
```

Comment:
This may be a bit of scope creep, but I wonder if our param passing usage is outliving it's historical utility.  Would it be worth creating a small wrapper class that can read the config and allow us to simplify the param passing here?

Reply:
The argument is more or less that it would allow us to unit-test the config expectations more clearly and when/if we want the YAML schema to change it would honor the interface to what uses it.

Reply:
I agree with you, but I am making this PR purely for "maintenance" purposes at this time. 

I think this entire program/script should be replaced/re-written using a more modern tech stack (likely serverless/lambdas) which would bring many benefits.